net.lutris.Lutris.yml

```diff
@@ -14,9 +14,11 @@ finish-args:
   - --share=network
   - --allow=multiarch
   - --allow=devel
+  - --talk-name=org.freedesktop.Flatpak.*
   - --talk-name=org.freedesktop.Notifications
   - --talk-name=org.gnome.Mutter.DisplayConfig
   - --filesystem=~/Games:create
+  - --filesystem=/tmp
```

Comment:
Do we need this? There is a `tmp` inside the sandbox.

Reply:
Yes, because the wrapper starts fusermount on the host.
`exec flatpak-spawn --host --forward-fd=1 --forward-fd=2 --forward-fd=3 $FD_ARGS fusermount "$@"`
This means the Appimage files get mounted to /tmp on the host and it needs to be mounted inside the sandbox to be accessible.

Reply:
@A6GibKm Should we also not include this per default and add it to the readme?